boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -0,0 +1,107 @@
+.. zephyr:board:: pocketbeagle_2
+
+Overview
+********
+
+PocketBeagle 2 is a computational platform powered by TI AM62x SoC (there are two
+revisions, AM6232 and AM6254).
+
+The board configuration provides support for the ARM Cortex-M4F MCU core and the
+following features:
+
+- Nested Vector Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+
+The board configuration also enables support for the semihosting debugging console.
```

Comment:
Where? why? if you already have a UART console.

Reply:
Well, this is what TI seems to have in the docs for [SK-AM64](https://docs.zephyrproject.org/latest/boards/ti/sk_am64/doc/index.html) and [SK-AM62](https://docs.zephyrproject.org/latest/boards/ti/sk_am62/doc/index.html). I think the generic stuff like UART and GPIO is supposed to be present in Supported Features, although that does not seem to be generated right now.